test/quantization/test_quantized_tensor.py

```diff
@@ -103,6 +103,36 @@ def test_qtensor(self):
                                  "quantization_scheme=torch.per_tensor_affine, " +
                                  "scale=1.0, zero_point=2)")
 
+    def test_qtensor_sub_byte(self):
+        num_elements = 10
+        scale = 1.0
+        zero_point = 2
```

Comment:
For 4 bit and 2 bit tensors, we should plan on having floating point scale and zero-point, as this format is primarily for lower precision storage.